apkid/rules/elf/obfuscators.yara

```diff
@@ -110,10 +110,12 @@ rule ollvm_v6_0_strenc : obfuscator
     // "Obfuscator-LLVM clang version 6.0.0 (trunk) (based on Obfuscator-LLVM 6.0.0git-b9ea5776)"
     $clang_version = "Obfuscator-LLVM clang version 6.0."
     $based_on      = "(based on Obfuscator-LLVM 6.0."
-    $strenc        = /datadiv_decode[0-9]{18,20}/
+    $strenc        = /\.datadiv_decode[\d]{18,20}/
 
   condition:
-    all of them
+    is_elf and
+    all of them and
```

Comment:
This is probably a nitpik - but I don't think this line should be `all of them` as this requires all the strings to be found in the binary, where we are explicitly checking below for the regex to be found in the symtab entries.

Reply:
2 of them?